examples/experimental/i3s-17-and-debug/bounding-volume-layer.js

```diff
@@ -3,7 +3,7 @@ import {Ellipsoid} from '@math.gl/geospatial';
 import {CubeGeometry, SphereGeometry} from '@luma.gl/engine';
 import {CompositeLayer, COORDINATE_SYSTEM, log} from '@deck.gl/core';
 import MeshLayer from './mesh-layer/mesh-layer';
-import {ORIENTED_BOUNDING_BOX, BOUNDING_SPHERE} from './constants';
+import {BOUNDING_VOLUME_MESH_TYPE} from './constants';
```

Comment:
Is just `BOUNDING_VOLUME_TYPE` enough?

Reply:
`BOUNDING_VOLUME_TYPE` is already in use. This is the opposite map for `cubeMesh` and `sphereMesh`.
![image](https://user-images.githubusercontent.com/70207219/120674815-26839800-c49d-11eb-909d-ee8dc13ee7c6.png)

Reply:
OK. It seemed to me that you just need an enum that distinguishes the types, and it is the same types in both cases. But this works too.